monkey/monkey_island/monkey_island.spec

```diff
@@ -0,0 +1,93 @@
+# -*- mode: python -*-
```

Comment:
This file is not PEP8

Reply:
except for a few (not so) long lines it is